### DISCUSSION TITLE
Breeze deferred css load issues with latest breeze version

### DISCUSSION THREAD

**Discussion by sandipansaha** [breezefront/community#319] [1. Help] [answered]
Hi @vovayatsyuk 

Recently we have noticed some anomalies regarding how the deferred CSS files are being loaded with latest Breeze versions.

We are using the following versions -

```
"name": "swissup/module-breeze",
"version": "2.17.11",

"name": "swissup/breeze-blank",
"version": "2.8.4",

"name": "swissup/theme-frontend-breeze-blank",
"version": "2.8.4",
```

Our site - https://master-7rqtwti-vcte4ksdazidy.eu-4.magentosite.cloud/breeze/
Here we have noticed that the deferred CSS files are not being deferred anymore. It is being loaded in page directly after the main CSS.
`<link rel="stylesheet" type="text/css" media="all" href="https://master-7rqtwti-vcte4ksdazidy.eu-4.magentosite.cloud/static/version1724159726/frontend/Swissup/breeze-blank/en_US/css/deferred-default.min.css">`

Whereas in Swissup demo site, it is being deferred loaded as expected.
`<link rel="stylesheet" media="print" onload="this.onload=null;this.media='all'" href="https://breeze.swissupdemo.com/static/version1714766523/frontend/Swissup/breeze-blank/en_US/css/deferred-default.min.css">`

I do not think that we did make any change related to that.
I was wondering whether it could be any bug that was introduced with any latest release recently? Or it would be some mistake that we have caused?
Could you please share the Breeze modules version from the Swissup demo?

Please enlighten us on this matter. Your help and feedback is appreciated as always.

Thanks
Sandipan

**Comment by vovayatsyuk** [comment, marked as answer]
Hello,

Our demo is outdated. Deferred CSS will be enabled when the `dev/css/use_css_critical_path` config is used. Otherwise, both CSS are now loaded normally.

This change was made due to not working deferred CSS at checkout when CSP is enabled.

Commit: https://github.com/breezefront/module-breeze/commit/a8ae95a4e6a63c4ce7811cf809fd981b9de1646f
